scripts/nix-profile.sh.in

```diff
@@ -34,4 +81,7 @@ if [ -n "$HOME" ]; then
     elif [ -e "$NIX_LINK/etc/ca-bundle.crt" ]; then # old cacert in Nix profile
         export SSL_CERT_FILE="$NIX_LINK/etc/ca-bundle.crt"
     fi
+
+    export PATH="$NIX_LINK/bin:$NIX_LINK/sbin:$__savedpath"
```

Comment:
NIX_LINK is user-provided via the HOME variable, if HOME happens to have a space in the name that would be annoying.

OTOH, the quotes don't hurt and using them always is easier and less error-prone than needing to decide when to use them.